Tests/ReactiveSwiftTests/SignalProducerSpec.swift

```diff
@@ -1962,6 +1962,59 @@ class SignalProducerSpec: QuickSpec {
 				let result = producer.single()
 				expect(result?.value) == 1
 			}
+			
+			context("with interval") {
+				
+				it("should send values at the given interval until hitting the limitation") {
+					
+					let scheduler = TestScheduler()
+					var count = 0
+
+					let original = SignalProducer<Int, TestError> { observer, _ in
+						
+						if count < 2 {
+							scheduler.schedule { observer.send(value: count) }
+							scheduler.schedule { observer.send(error: .default) }
+						} else {
+							scheduler.schedule { observer.sendCompleted() }
+						}
+						count += 1
+					}
+
+					var values: [Int] = []
+					var completed = false
+					
+					original.retry(upTo: Int.max, interval: 1, on: scheduler)
+					.start { event in
```

Comment:
This indentation is off. The ReactiveSwift style is to have methods like this be indented one stop.